src/pages/reference/providers/gcp/configuration.mdx

```diff
@@ -28,6 +28,16 @@ gcp-project-id: my-gcp-project-id
 # https://en.wikipedia.org/wiki/List_of_tz_database_time_zones
 schedule-timezone: Australia/Sydney # Available since v0.27.0
 
+# Import existing GCP Resources
+# Currently only secrets are supported
+# Available since v1.4.0
+import:
+  # A map of nitric names to GCP Secret Manager names
+  secrets: # Available since v1.4.0
+    # In typescript this would import the provided secret reference for a secret declared as
+    # const mySecret = secret('my-secret');
+    my-secret: existing-secret
```

Comment:
worth noting that the secret MUST be in the same project as they project ID defined in this stack.

Reply:
```suggestion
import:
  # A map of nitric names to GCP Secret Manager names
  secrets: # Available since v1.4.0
    # NOTE: Imported secrets must exist in the same project as specified by gcp-project-id :point_up: 
    # In typescript this would import the provided secret reference for a secret declared as
    # const mySecret = secret('my-secret');
    my-secret: existing-secret
```